README.md

```diff
@@ -35,7 +35,12 @@ The stylesheets are installed in `$PREFIX/share/ocr-transform/xslt`.
 
```

Comment:
Yes, it is. Using `ocr-transform` is one possibility how to do the transformation based on the stylesheets, but it is also possible to use other tools, for example `xsltproc`. Those tools expect the name of a stylesheet as an argument, so it is important to know where the stylesheets can be found.